sprite.go

```diff
@@ -940,7 +940,7 @@ func (p *Sprite) playDefaultAnim() {
 				p.goAnimateInternal(animName, &anicopy, false)
 			}
 		}
-		if !isPlayAnim {
+		if resetCostume && !isPlayAnim {
```

Comment:
What if a sprite is configured with `costumeIndex: 1` in sprite config, then it calls `animate` to play some animation after initiation:

```go
onStart => {
  play "jump"
}
```

Which costume will be rendered after animation "jump"?

We expect it to be costume with index `1`. While with this PR, it seems that the costume with index `0` will be rendered.

Reply:
I think the problem is that the value of `defaultCostumeIndex` is not correct when sprite inited. If it is correct, we won't need to specify `resetCostume: false` in `awake`.